app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= presenter.attribute_to_html(:date_modified, label: t('hyrax.base.show.last_modified')) %>
```

Comment:
Why are you overriding this template?  This wasn't mentioned in the PR description. Was it intentional?